packages/artifacts/rollup.config.ts

```diff
@@ -16,6 +16,17 @@ const banner = `/**
  * @license ${pkg.license}
  * @see [Github]{@link ${pkg.homepage}}
 */`
+const bannerCli = `#!/usr/bin/env node
+
+/**
+ * @module ${pkg.name}
+ * @version ${pkg.version}
+ * @file ${pkg.description}
+ * @copyright Ethereum Foundation ${new Date().getFullYear()}
+ * @license ${pkg.license}
+ * @see [Github]{@link ${pkg.homepage}}
+*/
+`
```

Comment:
This string is the same as the one above, I guess we can re-use it.

Reply:
https://github.com/privacy-scaling-explorations/snark-artifacts/pull/118/commits/31c7bab085bcc3eab2d575dc3f287a7047329a92